osu.Game/Overlays/Dashboard/Home/News/HomeNewsPanel.cs

```diff
@@ -0,0 +1,158 @@
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    FillMode = FillMode.Fill,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Alpha = 0
+                })
+                {
+                    RelativeSizeAxes = Axes.Both
+                };
+
+                bg.OnLoadComplete += d => d.FadeIn(250, Easing.In);
+
+                TooltipText = "view in browser";
+                Action = () => host.OpenUrlExternally("https://osu.ppy.sh/home/news/" + post.Slug);
+
+                HoverColour = Color4.White;
+            }
+        }
+
+        private class Footer : HomeNewsPanelFooter
+        {
+            protected override float BarPading => 10;
+
+            public Footer(APINewsPost post)
+                : base(post)
+            {
+            }
+
+            protected override NewsPostDrawableDate CreateDate(DateTimeOffset date) => new Date(date);
+
+            protected override Drawable CreateContent(APINewsPost post) => new FillFlowContainer
+            {
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                Margin = new MarginPadding { Top = 5, Bottom = 10 },
+                Spacing = new Vector2(0, 10),
+                Direction = FillDirection.Vertical,
+                Children = new Drawable[]
+                {
+                    new NewsTitleLink(post),
+                    new TextFlowContainer(f =>
+                    {
+                        f.Font = OsuFont.GetFont(size: 12, weight: FontWeight.Regular);
+                    })
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Text = post.Preview
+                    }
+                }
+            };
+
+            private class Date : NewsPostDrawableDate
+            {
+                public Date(DateTimeOffset date)
+                    : base(date)
+                {
+                    Margin = new MarginPadding { Top = 10 };
+                }
+
+                protected override Drawable CreateDate(DateTimeOffset date, OverlayColourProvider colourProvider) => new FillFlowContainer
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Vertical,
+                    Anchor = Anchor.TopRight,
+                    Origin = Anchor.TopRight,
+                    Children = new Drawable[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.TopRight,
+                            Origin = Anchor.TopRight,
+                            Font = OsuFont.GetFont(weight: FontWeight.Bold), // using Bold since there is no 800 weight alternative
+                            Colour = colourProvider.Light1,
+                            Text = $"{date: dd}"
```

Comment:
nit: unnecessary space

```suggestion
                            Text = $"{date:dd}"
```